app/assets/stylesheets/_account.scss

```diff
@@ -28,3 +28,7 @@
     }
   }
 }
+
+input[disabled] {
```

Comment:
concerned this will leak into other parts of the application

Far too permissive of a selector